go.mod

```diff
@@ -282,6 +282,7 @@ replace (
 	go.etcd.io/etcd/pkg/v3 => go.etcd.io/etcd/pkg/v3 v3.5.0
 	go.etcd.io/etcd/raft/v3 => go.etcd.io/etcd/raft/v3 v3.5.0
 	go.etcd.io/etcd/server/v3 => go.etcd.io/etcd/server/v3 v3.5.0
+	gopkg.in/square/go-jose.v2 v2.5.1 => gopkg.in/square/go-jose.v2 v2.2.2
```

Comment:
this is for fixing conformance compile error:
```
...

Step 7/15 : RUN CGO_ENABLED=0 GO111MODULE=off ginkgo build -ldflags "-w -s -extldflags -static" -r /go/src/github.com/kubeedge/kubeedge/tests/e2e
 ---> Running in 071f0bf47090
Compiling e2e...
Failed to compile e2e:

# github.com/kubeedge/kubeedge/vendor/k8s.io/kubernetes/pkg/serviceaccount
src/github.com/kubeedge/kubeedge/vendor/k8s.io/kubernetes/pkg/serviceaccount/claims.go:93:27: cannot use jwt.NewNumericDate(now.Add(time.Duration(warnafter) * time.Second)) (type *jwt.NumericDate) as type jwt.NumericDate in assignment

The command '/bin/sh -c CGO_ENABLED=0 GO111MODULE=off ginkgo build -ldflags "-w -s -extldflags -static" -r /go/src/github.com/kubeedge/kubeedge/tests/e2e' returned a non-zero code: 1
make: *** [Makefile:368: image] Error 1
Error: Process completed with exit code 2.
```
https://github.com/kubeedge/kubernetes/blob/v1.23.15-kubeedge/pkg/serviceaccount/claims.go#L93
k8s vendor use v2.2.2 go-jose.v2,  if we use v2.5.1, it will report the above error

in go-jose function NewNumericDate changed the return value from value to pointer, and cause the above issue

Reply:
Looks good, is it possible to use `v2.2.2` directly in require part? @gy95

Reply:
seems not, require part specify a min version number, and go will auto upgrade it to v2.5.1, so we need specify it in replace part. WDYT @fisherxu @vincentgoat

Reply:
> go will auto upgrade it to v2.5.1

Ohh, is there any other dependency use ` v2.5.1`?

Reply:
Replace will work. It's dependent on third-party packages.

Reply:
$ go mod why gopkg.in/square/go-jose.v2
# gopkg.in/square/go-jose.v2
github.com/kubeedge/kubeedge/build/conformance/kubernetes
github.com/kubeedge/kubeedge/build/conformance/kubernetes.test
k8s.io/kubernetes/test/e2e/auth
k8s.io/kubernetes/plugin/pkg/admission/serviceaccount
k8s.io/kubernetes/pkg/serviceaccount
gopkg.in/square/go-jose.v2

Reply:
k8s also replace this package to v2.2.2
https://github.com/kubeedge/kubernetes/blob/761eff0b24b24b2e3f545026e03dc13a4216b397/go.mod#L483